src/server/api/controllers/user.controller.js

```diff
@@ -321,6 +321,16 @@ const createAdminBySuperUser = async (id, body) => {
   });
 };
 
+const gerUsersbyorganizationId = async organization_id => {
```

Comment:
Can you rename this "gerUsersbyorganizationId " to "getUsersByOrganizationId"

Reply:
@Swapna-Sahu thanks for mentioning this...